govuk_bank_holidays/bank_holidays.py

```diff
@@ -110,6 +110,16 @@ def get_holidays(self, division=None, year=None):
             holidays = filter(lambda holiday: holiday['date'].year == year, holidays)
         return list(holidays)
 
+    def get_known_holiday_date_set(self, division=None):
```

Comment:
Adding this method reduce some duplication and makes the rest of code more readable.

However, this seems to me like an implementation detail. I'd be wary of adding this to the public interface.
I know there is no such a thing as a private method but maybe we could rename to `_get_known_holiday_date_set()` (so that it's more explicit we're not promising this method will "always" be there, etc).

Reply:
i started with that actually, but i suppose there's no harm in making it public: the docs explicitly explain that it's an unordered set